lib/thesis/api_controller.ex

```diff
@@ -8,11 +8,16 @@ defmodule Thesis.ApiController do
 
   def assets(conn, _params), do: conn
 
-  def update(conn, %{"contents" => contents, "page" => page} = _params) do
+  def update(conn, %{"contents" => contents, "page" => page}) do
     :ok = store.update(page, contents)
     json conn, %{}
   end
 
+  def delete(conn, %{"path" => path}) do
+    :ok = store.delete(%{"slug" => path})
+    json conn, %{}
+  end
```

Comment:
No explicit error handling, yet. I'm not sure how imperative it'll be to have that. I'd like to see how Thesis works in the wild.